test/rfqt_test.ts

```diff
@@ -13,7 +13,8 @@ import * as _ from 'lodash';
 import 'mocha';
 import * as request from 'supertest';
 
-import { getAppAsync, getDefaultAppDependenciesAsync } from '../src/app';
+import { getAppAsync } from '../src/app';
+import { getDefaultAppDependenciesAsync } from '../src/runners/utils';
```

Comment:
Do we want to also delete the require cache for `rfqt_test.ts` and `rfq_blocked_address_utils_test.ts` as well?

Reply:
i don't think we need it. the tests pass with the code the way it is now. so don't want to add weird stuff that isn't really required.